packages/frontend/src/components/Game/GameBoard/Tile/style.scss

```diff
@@ -1,10 +1,17 @@
+@import '~styles/memory-snap-mixins';
+
 .tile {
-  position:relative;
-  overflow:hidden;
-  padding-bottom: 100%;
+  height: 100px;
+  width: 100px;
+  overflow: hidden;
+  @include breakpoint(md) {
+    height: 200px;
+    width: 200px;
+  }
+
   img {
+    object-fit: contain; /* keep aspect ratio */
     width: 100%;
-    height: auto;
-    position: absolute;
+    margin-bottom: 1rem;
   }
-}
\ No newline at end of file
+}
```

Comment:
Instead of adding styles to contain the photos and set the aspect ratio, there's an option for the Pexels API to fetched pictures that have a square orientation. I can add a ticket for this or you could add it to this ticket. 

```javascript
// packages/backend/app.js
const response = await pexelsClient.photos.search({
      query,
      per_page,
      orientation: 'square', // add this
    });
```